### PR TITLE
14 pathology--need to solve multiple ids linking problems

### PR DESCRIPTION
**Pull request by SenseCode**
many thanks.....;)

### CODE REVIEW

**Comment by vsmalladi** [comment]
Fixed the multiple ids problem., might have just overly complicated the schema, but it should work now.

**Comment by vsmalladi** [comment]
@SenseCode Simplified python code to be in pathology reports instead of surgery. Also now have that ass a calucation at pathology-reports facet if we need.

**Comment by SenseCode** [comment]
@vsmalladi so should we set up a .py file for pathology_report? You didn't change surgery accession to  path-Id though...
we can control customized facets in this file

File: src/encoded/schemas/pathology_report.json [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+                    "RON,favor RO",
+                    "Sarcomatoid,NOS",
+                    "Sarcoma",
+                    "MiT family translocation RCC",
+                    "Tubulocystic RCC",
+                    "Unclassified",
+                    "Nephroblastoma(Wilms tumor)",
+                    "Soft tissue neoplasm, NOS",
+                    "SDH deficient RCC"
+                ]
```
Comment (vsmalladi) [inline src/encoded/schemas/pathology_report.json:120]: Need to double check that this is the list to use.

File: src/encoded/schemas/pathology_report.json [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+                    "Present",
+                    "Absent",
+                    "Not applicable",
+                    "Not available"
+                ]
+        },
+        "sarcomatoid_percentage": {
+            "title": "Sarcomatoid Percentage",
+            "description": "Percentage of tumor that was sarcomatoid. ",
+            "type": "number"
```
Comment (vsmalladi) [inline src/encoded/schemas/pathology_report.json:136]: Real data has range. Should make this a string.

File: src/encoded/schemas/pathology_report.json [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+                  "Not available"
+              ]
+      },
+      "ajcc_version": {
+        "title": "AJCC Version",
+        "description": "AJCC Version used for staging",
+        "type": "string",
+        "enum": [
+                "6th edition",
+                "7th edition"
```
Comment (vsmalladi) [inline src/encoded/schemas/pathology_report.json:333]: Lets add 8th edition.

**Review by vsmalladi — APPROVED** [review]

**Pull request merged**